variables.tf

```diff
@@ -1,9 +1,52 @@
-########################################################################################################################
+##############################################################################
 # Input Variables
-########################################################################################################################
+##############################################################################
 
-#variable "my_variable" {
-#  type        = string
-#  description = "A description of my variable"
-#  default     = "default_value"
-#}
+variable "cluster_id" {
+  type        = string
+  description = "Id of the target IBM Cloud OpenShift Cluster"
+  nullable    = false
+}
+
+variable "operator_helm_release_namespace" {
+  type        = string
+  description = "Namespace to deploy the helm releases. Default to ibm-mq-operator helm release"
+  default     = "ibm-mq-operator"
+  nullable    = false
+}
+
+variable "add_ibm_operator_catalog" {
+  type        = bool
+  description = "Flag to configure the IBM Operator Catalog in the cluster before installing the IBM MQ Operator. Default is true"
```

Comment:
I would change to `Whether to configure the IBM Operator Catalog in the cluster before the IBM MQ operator is installed. Default is `true`.`

Reply:
I think its ok to keep it that way - both are conveying same message, isn't it?